### PR TITLE
ugh

### PR DESCRIPTION
**Pull request by superyuuki**

### CODE REVIEW

File: RapidReact/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -51,6 +53,15 @@ public void robotInit() {
...
     this.configureButtonBindings();
 
+    //Subsystem Initialize Loop
+    if (Robot.isSimulation()) {
+      this.robotSubsystems.add(new LogTestSubsystem(this.config));
+      this.robotSubsystems.add(new SimulatorTestSubsystem(this.config));
+    }
+
+    // Configure the button bindings
+    this.configureButtonBindings();
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/Robot.java]: This is being called above.

File: RapidReact/src/main/java/frc/robot/simulator/SimulatorTestSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    @Override
+    public void periodic() {
+        int num = counter.incrementAndGet();
+
+        if (num > 700) {
+            System.exit(0);
+        }
+    }
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/simulator/SimulatorTestSubsystem.java:29]: We should comment this method and the subsystem so other developers know why we have a subsystem that exists after 700 iterations.

File: RapidReact/src/main/java/frc/robot/subsystem/BitBucketsSubsystem.java [hunk 3]
```diff
@@ -3,17 +3,34 @@
...
+    private final List<BaseTalon> motors;
+    private final Logger logger;
     protected final Config config;
 
     protected BitBucketsSubsystem(Config config)
     {
         this.setName(this.getClass().getSimpleName());
 
+        this.motors = new ArrayList<>();
+        this.addMotorsToList();
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/subsystem/BitBucketsSubsystem.java:25]: we removed all this code in the current version.

Comment (cpostbitbuckets) [reply RapidReact/src/main/java/frc/robot/subsystem/BitBucketsSubsystem.java:25]: i.e. we are no longer adding motors to the subsystem.

**Review by cpostbitbuckets — CHANGES_REQUESTED** [review]

File: RapidReact/src/main/java/frc/robot/simulator/SimulatorTestSubsystem.java (hunk 2 above)
Comment (superyuuki) [reply RapidReact/src/main/java/frc/robot/simulator/SimulatorTestSubsystem.java:29]: Comments are for losers

**Review by cpostbitbuckets — APPROVED** [review]

**Pull request merged**